packages/inventory/src/containers/ProfileView/tests/index.test.tsx

```diff
@@ -0,0 +1,182 @@
+  opensrpBaseURL,
+  history,
+  location: {
+    hash: '',
+    pathname: `${INVENTORY_SERVICE_POINT_PROFILE_VIEW}`,
+    search: '',
+    state: {},
+  },
+  match: {
+    isExact: true,
+    params: { [INVENTORY_SERVICE_POINT_PROFILE_PARAM]: 'b8a7998c-5df6-49eb-98e6-f0675db71848' },
+    path: `${INVENTORY_SERVICE_POINT_PROFILE_VIEW}`,
+    url: `${INVENTORY_SERVICE_POINT_PROFILE_VIEW}`,
+  },
+};
+
+store.dispatch(
+  authenticateUser(
+    true,
+    {
+      email: 'bob@example.com',
+      name: 'Bobbie',
+      username: 'RobertBaratheon',
+    },
+    // eslint-disable-next-line @typescript-eslint/camelcase
+    { api_token: 'hunter2', oAuth2Data: { access_token: 'bamboocha', state: 'abcde' } }
+  )
+);
+
+describe('Profile view Page', () => {
+  afterEach(() => {
+    fetch.resetMocks();
+    store.dispatch(deforest());
+  });
+
+  it('renders correctly', async () => {
+    fetch.once(JSON.stringify([])).once(JSON.stringify([])).once(JSON.stringify([]));
+
+    const wrapper = mount(
+      <Provider store={store}>
+        <Router history={history}>
+          <ServicePointProfile {...props}></ServicePointProfile>
+        </Router>
+      </Provider>
+    );
+
+    /** loading view */
+    expect(toJson(wrapper.find('.ant-spin'))).toBeTruthy();
+
+    await act(async () => {
+      await new Promise((resolve) => setImmediate(resolve));
+      wrapper.update();
+    });
+
+    expect(fetch.mock.calls.map((call) => call[0])).toEqual([
+      'https://test-example.com/rest/location/getAll?serverVersion=0&is_jurisdiction=false',
+      'https://test-example.com/rest/location/findByProperties?is_jurisdiction=true&return_geometry=false&properties_filter=status:Active,geographicLevel:0',
+      'https://test-example.com/rest/stockresource/servicePointId/b8a7998c-5df6-49eb-98e6-f0675db71848',
+    ]);
+    store.dispatch(removeLocationUnits());
+  });
+
+  it('renders when data is present', async () => {
+    fetch
+      .once(JSON.stringify(structures))
+      .once(JSON.stringify([madagascar]))
+      .once(JSON.stringify(inventories));
+
+    const wrapper = mount(
+      <Provider store={store}>
+        <Router history={history}>
+          <ServicePointProfile {...props}></ServicePointProfile>
+        </Router>
+      </Provider>
+    );
+
+    /** loading view */
+    expect(toJson(wrapper.find('.ant-spin'))).toBeTruthy();
+    await act(async () => {
+      await new Promise((resolve) => setImmediate(resolve));
+      wrapper.update();
+    });
+
+    // check fetch calls made
+    expect(fetch.mock.calls).toEqual(fetchCalls);
+  });
+
+  it('branch test for getNodePath with parentId undefined', async () => {
+    const sampleLocation = {
+      ...structure2,
+      properties: { ...structure2.properties, parentId: undefined },
+    };
+    const res = getNodePath(sampleLocation, [generateJurisdictionTree(madagascarTree)]);
+    expect(res).toEqual([]);
+  });
+
+  it('branch test for getNodePath with parentId', async () => {
```

Comment:
We should move this test case to `utils.test.tsx` since `getNodePath is from './utils`